[Content] Logo component documentation

ronijaakkola opened the pull request:
## Description
Adds documentation for the Logo component.

## How Has This Been Tested?
Tested by running the documentation site locally.

eemelinieminen commented on site/docs/components/logo.mdx:
```diff
@@ -0,0 +1,92 @@
+
+## Accessibility
+- A text alternative must always be provided for logos using the WAI-ARIA definition _aria-label_ or _aria-labelledby_. See <Link href="https://www.w3.org/TR/WCAG21/#text-alternatives" external>WCAG 2.1 Text Alternatives guideline</Link> for more information.
+- The colour contrast between the logo and its background must comply with <Link href="https://www.w3.org/TR/WCAG21/#contrast-minimum" external>AA Level contrast ratios</Link>.
```
Tähän vois vielä lisätä "more information" -linkin väri-dokumentaatioon. Vai meneekö liialliseksi toistoksi?

ronijaakkola replied:
Lisätty viite väridokumentaatioon.

ronijaakkola commented:
Changed the documentation so this can be released before Logo is implemented for Core. Removed Core code examples and links to Core Storybook. These can be added in later when Logo is implemented for Core.

eemelinieminen commented on site/docs/components/logo.mdx:
```diff
@@ -0,0 +1,82 @@
+See the [logo visual asset documentation](/visual-assets/logo "City of Helsinki logo") for more information about principles of using the City of Helsinki logo.
+
+## Principles
+- Logo is automatically included in HDS Navigation and HDS Footer (not yet released) components. **Use the Logo component sparingly in other parts of the service.**
```
Pitää muistaa ottaa toi (not yet released) sit pois myöhemmin :)

eemelinieminen commented on site/docs/components/logo.mdx:
```diff
@@ -0,0 +1,82 @@
+
+## Principles
+- Logo is automatically included in HDS Navigation and HDS Footer (not yet released) components. **Use the Logo component sparingly in other parts of the service.**
+- **It is recommended to include the service name next to the logo.** This helps the user more easily identify which city service they are currently using.
```
Tästä tuli mieleen, että pitääkö palvelun nimen tyylistä ja koosta suhteessa logoon antaa jotain ohjeistuksia? 

Nyt näkyy jo vähän poikkeavia toteutuksia naveissa. Toisaalta navilla ja footerilla tämä tietty yhtenäistyy, joten ei ehkä ole tarpeen.

ronijaakkola replied:
Jos sen haluaisi ohjeistaa, niin se pitäisi mielestäni toteuttaa sisään tähän komponenttiin. Ei sitä varmaan kukaan huomaa jos sen vain tänne kirjoittaa.

eemelinieminen reviewed (approved):
Hyvältä näyttää. Pari mieleen noussutta hajatelmaa kirjasin, mitkä ei sinänsä estä hyväksymistä.
